Assets/Scroll.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
```

Comment:
out of curiosity, what do these summary tags do? and is the triple /// important for this? makes me think of javadocs for some reason, just wondering if it might be the same thing.